README.md

```diff
@@ -14,6 +14,15 @@ An overview of the architecture can be found at [netplan.io/design](https://netp
 
 The full documentation for netplan is available in the [doc/netplan.md file](../master/doc/netplan.md)
 
+# Build using Meson
+
+Steps to build netplan using the [Meson](https://mesonbuild.com) build system inside the `build/` directory:
+
+* meson setup build --prefix=/usr -Db_coverage=true
+* meson compile -C build
+* meson test -C build --verbose [TEST_NAME]
+* meson install -C build --destdir ../tmproot
```

Comment:
question: Do we actually want the user to install binaries that were compiled with coverage support?

Reply:
No, absolutely not. This documentation is mostly given for development purposes. But you're right in that it can easily be mistaken. Let me put that `-Db_coverage=true` into square brackets to make people actually think about it (unable to blindly apply that command).